vault.js

```diff
@@ -1,4 +1,35 @@
+/*jshint esversion: 6 */
 'use strict';
-module.exports = function() {
-  
-};
\ No newline at end of file
+module.exports = function(){
+
+var vault = {};
+
+  function setValue(key, value){
+     vault[key] = value;
```

Comment:
Please check but it looks like your vault assignment is one character to the right.